### PR TITLE
Update the docs based on the release v0.2.0

### PR DESCRIPTION
**Pull request by tpiperatgod**
add doc for [code PR #361 ](https://github.com/streamnative/function-mesh/pull/361)
1. update docs for installing Function Mesh Operator with Helm
2. add the v0.2.0 release notes

Signed-off-by: laminar <tpiperatgod@gmail.com>

### CODE REVIEW

File: docs/releases/release-note-0-2-0.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+title: Release notes v0.2.0
+category: releases
+id: release-note-0-2-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.2.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.2.0).
+
+## Support for the Pulsar stateful functions
+
+In this release, we have added `Stateful`, `PulsarStateStore` resources, and `--state_storage_serviceurl`, `-state_storage_impl_class` parameters to support Pulsar's stateful functions.
```
Comment (freeznet) [inline docs/releases/release-note-0-2-0.md:11]:
```suggestion
In this release, we have added `Stateful`, `PulsarStateStore` resources, and `--state_storage_serviceurl`, `-state_storage_impl_class` parameters to support running Pulsar's stateful functions.
```

**Review by freeznet — APPROVED** [review]

File: docs/install-function-mesh.md [hunk 2]
```diff
@@ -43,72 +43,78 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
...
-    git clone https://github.com/streamnative/function-mesh.git
-    cd function-mesh
+    helm repo add function-mesh http://charts.functionmesh.io/
+    helm repo update
     ```
 
-2. Create the custom resource type.
+2. Install the Function Mesh Operator.
+
+    Let's set some variables for convenient use later.
```
Comment (Huanli-Meng) [inline docs/install-function-mesh.md:55]: I know setting the environment variable is a convenient way. But maybe it's not suitable for all users.

File: docs/install-function-mesh.md [hunk 3]
```diff
@@ -43,72 +43,78 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
...
 
-3. Install the Function Mesh Operator.
-
-   1. Create a Kubernetes namespace `function-mesh`. If no Kubernetes namespace is specified, the `default` namespace is used.
+    Install the Function Mesh Operator via following command.
 
-        ```shell
-        kubectl create ns function-mesh
-        ```
+    > If no Kubernetes namespace is specified, the `default` namespace is used.
```
Comment (Huanli-Meng) [inline docs/install-function-mesh.md:64]:
```suggestion
    > **Note**
    >
    > - If no Kubernetes namespace is specified, the `default` namespace is used.
```

File: docs/install-function-mesh.md [hunk 4]
```diff
@@ -43,72 +43,78 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
...
+    >
+    > If the namespace ${FUNCTION_MESH_RELEASE_NAMESPACE} doesn't exist yet, you can add the parameter `--create-namespace ` to create it automatically.
```
Comment (Huanli-Meng) [inline docs/install-function-mesh.md:66]:
```suggestion
    > - If the namespace ${FUNCTION_MESH_RELEASE_NAMESPACE} doesn't exist yet, you can add the parameter `--create-namespace ` to create it automatically.
```

File: docs/install-function-mesh.md [hunk 5]
```diff
@@ -43,72 +43,78 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
...
-        | --- | --- | --- |
-        |`enable-leader-election`| Whether the Function Mesh Controller Manager should enable leader election. | true|
-        | `enable-pprof` |Whether the Function Mesh Controller Manager should enable [pprof](https://github.com/google/pprof). | `false`|
-        |`pprof-addr`|The address of the pprof. |`:8090`|
-        | `metrics-addr`| The address of the metrics. |`:8080`|
-        | `health-probe-addr`|The address of the health probe. |`:8000`|
-        |`config-file`| The configuration file of the Function Mesh Controller Manager, which includes `runnerImages`, `resourceLabels`, and `resourceAnnotations` configurations. <br />- `runnerImage`: the runner image to run the Pulsar Function instances. Currently, it supports Java, Python, and Go runner images. <br />- `resourceLabels`: set labels for Pulsar Functions, Sources, or Sinks. <br />- `resourceAnnotations`: set annotations for Pulsar Functions, Sources, or Sinks.  |`/etc/config/configs.yaml`|
+    | Parameters | Description | Default|
+    | --- | --- | --- |
+    |`enable-leader-election`| Whether the Function Mesh Controller Manager should enable leader election. | true|
```
Comment (Huanli-Meng) [inline docs/install-function-mesh.md:76]:
```suggestion
    |`enable-leader-election`| Whether the Function Mesh Controller Manager should enable leader election. | `true` |
```

File: docs/releases/release-note-0-2-0.md [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+title: Release notes v0.2.0
+category: releases
+id: release-note-0-2-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.2.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.2.0).
+
+## Support for the Pulsar stateful functions
+
+In this release, we have added `Stateful`, `PulsarStateStore` resources, and `--state_storage_serviceurl`, `-state_storage_impl_class` parameters to support running Pulsar's stateful functions.
```
Comment (Huanli-Meng) [inline docs/releases/release-note-0-2-0.md:11]:
```suggestion
In this release, we added the following resources and parameters to run Pulsar stateful functions.
- Resources: `Stateful` and `PulsarStateStore`
- Parameters: `--state_storage_serviceurl` and `-state_storage_impl_class`.
```

File: docs/releases/release-note-0-2-0.md [hunk 7]
```diff
@@ -0,0 +1,20 @@
+---
+title: Release notes v0.2.0
+category: releases
+id: release-note-0-2-0
+---
+
+Here are some highlights of this release. For a full list of updates available for Release v0.2.0, check out the [Function Mesh change log](https://github.com/streamnative/function-mesh/releases/tag/v0.2.0).
+
+## Support for the Pulsar stateful functions
```
Comment (Huanli-Meng) [inline docs/releases/release-note-0-2-0.md:9]:
```suggestion
## Support Pulsar stateful functions
```

File: docs/releases/release-note-0-2-0.md [hunk 8]
```diff
@@ -0,0 +1,20 @@
...
+
+In this release, we have added `Stateful`, `PulsarStateStore` resources, and `--state_storage_serviceurl`, `-state_storage_impl_class` parameters to support running Pulsar's stateful functions.
+
+## Independent the helm chart of Function Mesh Operator
+
+In this release, we have set up the dedicated helm charts repository for the Function Mesh Operator. It is now maintained through the domain https://charts.functionmesh.io. You can find information about the repository [here](https://artifacthub.io/packages/helm/function-mesh/function-mesh-operator).
```
Comment (Huanli-Meng) [inline docs/releases/release-note-0-2-0.md:15]:
```suggestion
In this release, we set up a dedicated Helm charts repository for the Function Mesh Operator. It is now maintained through this [domain](https://charts.functionmesh.io). For details, see [Function Mesh Operator repository](https://artifacthub.io/packages/helm/function-mesh/function-mesh-operator).
```

File: docs/releases/release-note-0-2-0.md [hunk 9]
```diff
@@ -0,0 +1,20 @@
...
+
+## Support Golang 1.18
+
+In this release, we have added support for Golang 1.18.
```
Comment (Huanli-Meng) [inline docs/releases/release-note-0-2-0.md:19]:
```suggestion
In this release, Function Mesh supported Golang 1.18.
```

**Review by Huanli-Meng — COMMENTED** [review]
Leave some comments. PTAL

**Review by Huanli-Meng — APPROVED** [review]
LGTM

**Pull request merged**